jobserver/views/jobs.py

```diff
@@ -64,17 +65,25 @@ def get(self, request, *args, **kwargs):
         # have more than one backend-specific error code it makes sense to keep
         # things together.
         log_path = ""
+        log_path_url = ""
         if job.job_request.backend.slug == "tpp" and job.status_code == "nonzero_exit":
             log_path = os.path.join(
                 job.job_request.backend.parent_directory,
                 job.job_request.workspace.name,
                 "metadata",
                 f"{job.action}.log",
             )
+            log_path_url = (
+                furl(job.job_request.workspace.get_files_url())
+                / "metadata"
+                / f"{job.action}.log"
+            )
+            log_path_url.path.normalize()
 
         context = {
```

Comment:
Do we need this because `get_files_url()` returns a trailing slash?

Reply:
Yes, unfortunately.